src/Data/Neuroglia.Data/Neuroglia.Data.csproj

```diff
@@ -40,4 +40,9 @@
   <ItemGroup>
     <Folder Include="Properties\" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
Why did you add LocalizedResources.resx to the csproj? Wasn't the case before

Reply:
Apparently it wasn't.
I think this happens when I change its access modifier from `no code gen` to `public` to regenerate  the `LocalizedResources.Designer.cs` the switch it back to `no code gen`.
So I think this should be removed.